vulture/core.py

```diff
@@ -384,6 +384,14 @@ def visit_FunctionDef(self, node):
             if param and isinstance(param, str):
                 self._define_variable(param, node.lineno, confidence=100)
 
+    def visit_If(self, node):
+        if utils.condition_is_unsatisfiable(node.test):
+            self._define(self.unreachable_code, 'if', node.lineno,
+                         size=lines.count_lines(node) if self.sort_by_size
+                         else 1,
+                         message="unsatisfiable 'if' condition",
+                         confidence=100)
+
```

Comment:
You will probably need to convert is_condition_unsatisfiable() to evaluate_condition() and let it return True, False or raise ValueError if the condition cannot be evaluated by ast.literal_eval().